code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -84,6 +84,21 @@
 	var/registered_name = null // The name registered_name on the card
 	var/assignment = null
 	var/dorm = 0		// determines if this ID has claimed a dorm already
+	var/disabled = FALSE // if it has been remotely disabled
+	var/tier = 1 // level of the card for disabling purposes

```

Comment:
Tiers are checked by comparing the values, and 4 < 1 is more clear than TIER_BASIC < TIER_CENTCOMM